src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -482,8 +482,8 @@ protected function _bundleTranslatedFields($entity)
      */
     protected function _findExistingTranslations($ruleSet)
     {
-        $target = TableRegistry::get($this->_config['translationTable']);
-        $association = $this->_table->association($target->alias());
+        $targetAlias = Inflector::slug($this->_config['translationTable']);
```

Comment:
The latter better.
slug() should only be used when the input can be non alpha numeric as it does a lot of overhead.